MUXSDKStats/MUXSDKStatsTests/MUXSDKStatsTests.m

```diff
@@ -96,6 +137,25 @@ - (void)testVideoChangeForAVPlayerLayer{
     [MUXSDKStats destroyPlayer:playName];
 }
 
+- (void)testProgramChangeForAVPlayerViewController{
+    MuxMockAVPlayerViewController *controller = [[MuxMockAVPlayerViewController alloc] init];
+    MUXSDKCustomerPlayerData *customerPlayerData = [[MUXSDKCustomerPlayerData alloc] initWithEnvironmentKey:@"ENV_KEY"];
+    MUXSDKCustomerVideoData *customerVideoData = [[MUXSDKCustomerVideoData alloc] init];
+    [customerVideoData setVideoTitle:@"01234"];
+    NSString *playName = @"Player";
+    MUXSDKPlayerBinding *playerBinding = [MUXSDKStats monitorAVPlayerViewController:controller withPlayerName:playName playerData:customerPlayerData videoData:customerVideoData];
+    XCTAssertNotNil(playerBinding, "expected monitorAVPlayerViewController to return a playerBinding");
+    [customerVideoData setVideoTitle:@"56789"];
+    [MUXSDKStats programChangeForPlayer:playName withVideoData:customerVideoData];
+    NSArray *expectedEventTypes = @[MUXSDKPlaybackEventViewInitEventType,
```

Comment:
Shouldn't we have more events here?
```
// monitor
MUXSDKPlaybackEventViewInitEventType,
MUXSDKDataEventType,
MUXSDKPlaybackEventPlayerReadyEventType,

// program change
MUXSDKPlaybackEventViewEndEventType
MUXSDKPlaybackEventViewInitEventType,
MUXSDKDataEventType,
MUXSDKPlaybackEventPlayerReadyEventType,
MUXSDKPlaybackEventPlayEventType
MUXSDKPlaybackEventPlayingEventType
```

Reply:
@nidhik good catch! I just updated and updated the asserts with some more friendly error messages and also assert that the number of events matches the expected

Reply:
Thanks 😄